packages/context/src/context-view.ts

```diff
@@ -158,14 +163,23 @@ export class ContextView<T = unknown> extends EventEmitter
  * Create a context view as a getter
  * @param ctx Context object
  * @param bindingFilter A function to match bindings
+ * @param bindingSorter A function to sort matched bindings
  * @param session Resolution session
  */
 export function createViewGetter<T = unknown>(
   ctx: Context,
   bindingFilter: BindingFilter,
+  bindingSorterOrSession?: BindingSorter | ResolutionSession,
   session?: ResolutionSession,
```

Comment:
This API allows the following invocation which does not seem valid to me:

```ts
createViewGetter<T>(ctx, filter, session1, session2);
```

Please use function overloads to better describe the actual API.

```ts
export function createViewGetter<T = unknown>(
  ctx: Context,
  bindingFilter: BindingFilter,
  bindingSorter: BindingSorter,
  session?: ResolutionSession,
): Getter<T[]>;

export function createViewGetter<T = unknown>(
  ctx: Context,
  bindingFilter: BindingFilter,
  session?: ResolutionSession,
): Getter<T[]>;

export function createViewGetter<T = unknown>(
  ctx: Context,
  bindingFilter: BindingFilter,
  bindingSorterOrSession?: BindingSorter | ResolutionSession,
  session?: ResolutionSession,
): Getter<T[]> {
  // the implementation
}
```